server/app/controllers/InternalServerErrorController.java

```diff
@@ -0,0 +1,34 @@
+package controllers;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import javax.inject.Inject;
+import javax.inject.Provider;
+import play.i18n.MessagesApi;
+import play.mvc.Controller;
+import play.mvc.Http;
+import play.mvc.Result;
+import play.mvc.Results;
+import views.errors.InternalServerError;
+
+public class InternalServerErrorController extends Controller {
```

Comment:
**suggestion:** I think we have a mix of doing this and not, but could we add a javadoc for this class?